base.sh

```diff
@@ -477,14 +464,15 @@ myecho
 myecho "BEGIN base $%(CSUB_JOBID)s `date`"
 myecho
 
-# check whether BLCR support is available
-/sbin/lsmod | grep blcr >& /dev/null
-if [ $? -ne 0 ]
-then
-	myecho "No BLCR support available here (`hostname`), aborting job."
-	endjob
-	exit 30
-fi
+# check whether DMTCP commands are available
+for cmd in $DMTCP_COMMAND $DMTCP_LAUNCH $DMTCP_RESTART; do
+    which $cmd > /dev/null
```

Comment:
testing one is ok; for blcr, you needed to check if a module was actually loaded, so a bit different.